test/src/e2e_vm_tests/test_programs/should_pass/dca/struct_field_no_warning/test.toml

```diff
@@ -0,0 +1 @@
+category = "compile"
```

Comment:
Are we meant to say that we expect no warnings here via `expected-warnings`? Or is this implied?

Reply:
Ah nice. Thanks!